### PR TITLE
LPS-71422 Fixes accessibility issue in IE11 caused by SVG icons

### PR DESCRIPTION
**Pull request by alexandervalencia**

### CODE REVIEW

**Comment by peterborkuti** [comment]
ci:retest

**Comment by Robert-Frampton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by Robert-Frampton** [comment]
I think this PR shows most of the instances we need to change.

https://github.com/brianchandotcom/liferay-portal/pull/48453/commits/4ad525834e182810d541954e5dfce0b70c8c4481

**Comment by alexandervalencia** [comment]
Further fixes to additional files requiring 'focusable="false"' attribute on SVG tags.

File: util-taglib/src/com/liferay/taglib/aui/ATag.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ protected int processEndTag() throws Exception {
 
 				jspWriter.write(StringPool.SPACE);
 				jspWriter.write("<svg class=\"lexicon-icon ");
-				jspWriter.write("lexicon-icon-shortcut\" role=\"img\"><use ");
+				jspWriter.write("focusable=\"false\" lexicon-icon-shortcut\" role=\"img\"><use ");
```
Comment (Robert-Frampton) [inline util-taglib/src/com/liferay/taglib/aui/ATag.java]: Looks like you put this inside the `class`.

File: modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-;(function(A, $, _, Liferay) {
+d;(function(A, $, _, Liferay) {
```
Comment (Robert-Frampton) [inline modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util.js]: Make sure to look over your changes or use linters to catch accidental changes :)

**Comment by alexandervalencia** [comment]
Thanks for your patience. Corrections made.

**Comment by Robert-Frampton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by Robert-Frampton** [comment]
ci:retest

**Comment by Robert-Frampton** [comment]
Hey @alexandervalencia 

I went ahead and sent to Brian https://github.com/brianchandotcom/liferay-portal/pull/48598.

Take not of the commit names, they should all start with the JIRA ticket (I went ahead and updated them). The source formatting commit is due to the line going over 80 characters long, which is a standard we use for java files in Portal.

**Pull request closed without merging**